src/Source/DoctrineORMQuerySourceIterator.php

```diff
@@ -25,17 +25,18 @@ final class DoctrineORMQuerySourceIterator extends AbstractPropertySourceIterato
     public function __construct(
         Query $query,
         array $fields,
-        string $dateTimeFormat = 'r',
+        ?string $dateTimeFormat = null,
         private int $batchSize = 100,
-        bool $useBackedEnumValue = true,
+        ?bool $useBackedEnumValue = null,
+        bool $disableSourceFormatters = true
```

Comment:
Why is it false by default  in the abstract and true by default here ?

Reply:
My bad, thanks (I guess it was preventing the new deprecations).
I'll be updating this soon.